jboss/container/wildfly/launch/jgroups/test/jgroups.bats

```diff
@@ -195,8 +194,8 @@ EOF
 
     run configure_jgroups_encryption
     echo "${output}"
-    [[ "${output}" =~ "INFO Configuring JGroups cluster traffic encryption protocol to SYM_ENCRYPT." ]]
-    [[ "${output}" =~ "WARN Detected missing JGroups encryption configuration, the communication within the cluster WILL NOT be encrypted." ]]
+    [[ "${output}" =~ .*"INFO Configuring JGroups cluster traffic encryption protocol to SYM_ENCRYPT.".* ]]
```

Comment:
Same regex comment as above, I won't mention all of these, but something to be aware of.